### DISCUSSION TITLE
Проблемы шейдинга моделей на некоторых составах

### DISCUSSION THREAD

**Discussion by CralixRaev** [metrostroi-repo/MetrostroiAddon#204] [General]
На некоторых составах в некоторых местах есть очевидные беды с освещением, которые возникли цитата "из за хакнутого компилятора". В данных сломанных моделях сама модель расклеена на множество бадигруп со странным делением. Чинится обычным декомпилом -> объединением всего -> исправления сглаживания -> рекомпиляцией. На скринах ниже показываю результат до и после
![image](https://user-images.githubusercontent.com/32234868/104169186-36c4a900-5410-11eb-8225-c1a3097e29f6.png)
![image](https://user-images.githubusercontent.com/32234868/104169239-4c39d300-5410-11eb-9b22-622b265c1bff.png)
а вот что мы можем увидеть в редакторе
![image](https://user-images.githubusercontent.com/32234868/104169303-670c4780-5410-11eb-9800-6e1601fe36c7.png)
![image](https://user-images.githubusercontent.com/32234868/104169312-696ea180-5410-11eb-826c-0701790edad9.png)
такая история не только на ТИСУ, а так же и на некоторых Е-образных и многих других.

**Comment by old702** [comment]
Заметка: Рекомпилить в DMX формат!

**Comment by CrIcKeT98** [comment]
DMX тут не нужен, проблема лишь в том как этот сторонний компилятор определяет боди-группы. В 760 все ок 
![image](https://user-images.githubusercontent.com/33666353/104223717-09e6b500-5455-11eb-9015-f61459cf5009.png)

**Comment by cyber-dream** [comment]
В dmx нужно все перевести, это кучу ошибок уберет, которые мы даже заметить немогли. Пошагово на таски раскидаю

**Comment by Alexrig23** [comment]
будут ли в обнове метростроя исправлены полигоны с мордой Еж/Еж1 и Еж3/Ем-508Т?

**Comment by cyber-dream** [comment]
Да

**Comment by Alexrig23** [comment]
![Снимок экрана (712)](https://user-images.githubusercontent.com/80965105/113826843-4c393d00-978b-11eb-80e9-8de6d4be88b1.png)
![Снимок экрана (711)](https://user-images.githubusercontent.com/80965105/113826846-4e030080-978b-11eb-9307-b9387d953e2b.png)

Я исправил модельки Еж3 с Ем-508Т и Еж/Еж1 но как убрать тень от тифона возле левого окошка где машинист?

**Comment by CralixRaev** [comment]
Очень похоже на то, что он довешивается потом
Я приду и попробую заняться уже этим полноценно

**Comment by Alexrig23** [comment]
> Очень похоже на то, что он довешивается потом
> Я приду и попробую заняться уже этим полноценно

Я уже с ними закончил, осталось только в игру перенести а я только забыл как компилить модельки SMD.

**Comment by Alexrig23** [comment]
> Очень похоже на то, что он довешивается потом
> Я приду и попробую заняться уже этим полноценно

Хотя стоп, а как всё же убрать тень от тифона??

**Comment by cyber-dream** [comment]
Ребят, давайте не брать таски, уже выданные другим людям. Чтобы не делать двойную работу.

**Comment by CralixRaev** [comment]
> > Очень похоже на то, что он довешивается потом
> > Я приду и попробую заняться уже этим полноценно
> 
> Я уже с ними закончил, осталось только в игру перенести а я только забыл как компилить модельки SMD.

жирно плюсую кибердрима. если человек уже назначен - значит он в процессе и знает больше чем ты, может, не нужно пытаться сделать его работу?

**Comment by cyber-dream** [comment]
Ну давайте без негатива только. От меня тоже прогресса не было по этому таску, нормальное желание самому сделать. Просто немножко формальностей упростят нам жизнь: сначала здесь что-то обсуждаем - потом тратим своё время на работу

**Comment by Alexrig23** [comment]
![Снимок экрана (754)](https://user-images.githubusercontent.com/80965105/114727871-41f8df00-9d47-11eb-8772-d07fbb345a29.png)
![Снимок экрана (755)](https://user-images.githubusercontent.com/80965105/114727879-43c2a280-9d47-11eb-9004-f247368ec3da.png)
![Снимок экрана (756)](https://user-images.githubusercontent.com/80965105/114727882-43c2a280-9d47-11eb-8fa3-75d957b751ef.png)
![Снимок экрана (751)](https://user-images.githubusercontent.com/80965105/114727886-445b3900-9d47-11eb-94aa-78cc76c30eb4.png)
![Снимок экрана (752)](https://user-images.githubusercontent.com/80965105/114727888-445b3900-9d47-11eb-87a4-6a029214386f.png)
![Снимок экрана (753)](https://user-images.githubusercontent.com/80965105/114727891-44f3cf80-9d47-11eb-9db0-d9cac28d0d79.png)

Вот прогресс, только на дефолтных скинах Ежа осталась тень от тифона возле окошки машика, на фанатских скинах такое не наблюдается.

**Comment by Alexrig23** [comment]
![Снимок экрана (759)](https://user-images.githubusercontent.com/80965105/114731656-98b3e800-9d4a-11eb-8f6a-a05785c6da04.png)
![Снимок экрана (760)](https://user-images.githubusercontent.com/80965105/114731666-9a7dab80-9d4a-11eb-8983-2098c2d8d8a9.png)

Морда Еж3/Ем-508Т и Еж/Еж1 - FIXED

**Comment by glebqip** [comment]
> Ребят, давайте не брать таски, уже выданные другим людям. Чтобы не делать двойную работу.

**Comment by old702** [comment]
Отличная работа
